src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a
+      className="UserInfo"
+      href={`mailto:${user?.email}`}
+    >
+      {user?.name}
+    </a>
+  );
+};
```

Comment:
According comment about this component after fixing code there always will be user object, so we can destructure it 🤭 

```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
  const { email, name } = user;

  return (
    <a
      className="UserInfo"
      href={`mailto:${email}`}
    >
      {name}
    </a>
  );
};
```